File: messages/src/main/java/com/thundashop/core/resturantmanager/ResturantTable.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.resturantmanager;

import com.thundashop.core.cartmanager.data.Cart;
import com.thundashop.core.common.DataCommon;
import java.util.ArrayList;
import java.util.List;
import org.mongodb.morphia.annotations.Transient;

/**
 *
 * @author ktonder
 */
public class ResturantTable extends DataCommon {
    public String table = "";
    
    public List<String> savedCartIds = new ArrayList();
    
    public String currentCartId = "";
            
    public String name = "";
    
    @Transient
    public Cart currentCart = null;
}

File: html5apps/terminal/events/core/resturantmanager/ResturantTable.php
<?php
class core_resturantmanager_ResturantTable extends core_common_DataCommon  {
	/** @var String */
	public $table;

	/** @var String */
	public $savedCartIds;

	/** @var String */
	public $currentCartId;

	/** @var String */
	public $name;

	/** @var core_cartmanager_data_Cart */
	public $currentCart;

}
?>